homeTask_4/homeTask_4/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+
+namespace homeTask_4
+{
+    class Program
+    {
+
+        
+//1. Создать два произвольных массива, подсчитать максимальное и минимальное значение их элементов.Вывести на экран.
+//2. Поменять в массивах максимальные и минимальные значения местами
+        static void InitializeArray(int[] array)
+        {
+            Random rnd = new Random();
+            for (int i = 0; i < array.Length; ++i)
+            {
+                array[i] = rnd.Next(1, 50);
+            }
+        }
+
+        static void PrintArray(int[] array)
+        {
+            for (int i = 0; i < array.Length; ++i)
+            {
+                Console.WriteLine("№={0} is {1}", i + 1, array[i]);
+            }
+        }
+
+        static int MaxElement(int[] array)
+        {
+            int max = array[0];
+            for (int i = 0; i < array.Length; ++i)
+                if (array[i] > max) 
+                 max = array[i];
+                    
+            return max;
+        }
+
+        static int MinElement(int[] array)
+        {
+            int min = array[0];
+            for (int i = 0; i < array.Length; i++)
+                if (array[i] < min)
+                   min = array[i];
+                 
+            return min;
+        }
+        static void SwitchArray(int[] array)
+        {
+            int min = array[0];
+            int max = array[0];
+            int minNumber = 0;
+            int maxNumber = 0;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] > max) {
+                    max = array[i];
+                    maxNumber = i;
+                        }
+                if (array[i] < min) {
```

Comment:
in this way we can use `if{}else{}` construction instead of using `if` twice.
or only `if`
or [ternary operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator)

Reply:
ok. I will try to use ternary operator.

Reply:
good choice

Reply:
Good evening @EgorAntonovich. I think that it is impossible to use  **if{}else{}** for this way. if
condition **if (array[i] < min)**is met, I haven't to use the "else" part bcz my condition is completed and I no longer need any "else". only if  I can write
e.g 

if (array[i] > max)
               max = array[i];
               else
                if (array[i] < min)
                 min = array[I];
...
but in this case in  **else if** constructions i check another  parameter(**min**) which has nothing to do with **max** 

I have 3 options for this solution:
1) 
if (array[i] > max)
               maxnumber= i;
               else
                if (array[i] < min)
                 minnumber = i;
...
2) current one

for (int i = 0; i < array.Length; i++)
            {
                if (array[i] > max) { ...}
                if (array[I] < min) { ...}
3) and ternary operator
maxNumber = (array[i] > max) ? i : maxNumber;
minNumber = (array[I] < min) ? i : minNumber ;
                    
Please correct me if I'm wrong

Reply:
You are right. You can use one of this varians which is better in your opinion. I approve everyone.